FreeRTOS/FreeRTOS
# Update coreHTTP S3 Download Instructions

Update demo instructions for coreHTTP S3 Download demo. 
Also, remove broken links from the comment blocks above the S3 Root CA macros in the Upload and Download Multithreaded demos
Description
-----------

Update the instructions in places that were either inaccurate or required further clarification to help resolve issues that come up when trying to configure and run the demo
Test Steps
-----------
Tested by following the updated demo instructions and was successfully able to complete the demo.


Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
